public/init_vic.json

```diff
@@ -11,7 +11,7 @@
             "name": "Victorian Government",
             "type": "ckan",
             "url": "http://www.data.vic.gov.au/data",
-            "filterQuery": "res_format:wms",
+            "filterQuery": ["fq=res_format%3awms"],
```

Comment:
Ckan queries can be quite extensive including urlencoded JSON objects.  It's seems all the different methods have strengths and weaknesses so I left it the most flexible.  Also there is a 'wonderful' search feature which won't allow us to search for BOTH esri resources and wms resources, so it needs to be an array to carry out multiple searches if necessary.